### ISSUE TITLE
MD5 Hash not matching for Remote

### ISSUE THREAD

**Issue by crispindev** [GoogleCloudPlatform/anthos-workshop#54]
There seems to be an error in the MD5 Hash:

```
Cluster.kops "remote.k8s.local" not found
Create kops cluster...
I0129 10:14:33.801998    1320 create_cluster.go:519] Inferred --cloud=gce from zone "us-central1-a"
$l/config: googleapi: Error 400: Provided MD5 hash "1B2M2Y8AsgTpgAmY7PhCfg==" doesn't match calculated MD5 hash "aP2excWSm858AztEo1StmA=="., invalid 

error writing updated configuration: error writing Cluster "remote.k8s.local": error writing configuration file gs://democol-adk-prod-kops-remote/remote.k8s.local/config
error: cannot locate context remote.k8s.local
Check if cluster is ready - Attempt 1
```

**Comment by KrzysztofZalasa** [comment]
It has been solved in new version of kops, I updated env file for     
`export KOPS_VERSION=v1.16.0`

**Comment by cgrant** [comment]
Closing as this was addressed in #60
